Automattic/pocket-casts-ios
# Filter out any empty URLs that may be returned from the OPML import process

Fixes: https://github.com/Automattic/pocket-casts-ios/issues/576

The server request can fail if we try to send a completely empty URL to it, so this filters out the empty url, and updates the Apple Podcasts import shortcut. 

## To test


```
<?xml version="1.0" encoding="utf-8" standalone="no"?>
<opml version="1.0">
	<body>
		<outline text="feeds">
			<outline xmlUrl="https://podcasts.apple.com/us/podcast/hello-from-the-magic-tavern/id975078285?uo=4" type="link" />
			<outline xmlUrl="" type="link" />
		</outline>
	</body>
</opml>
```

1. Save the above content as podcasts.opml file on your Desktop
2. Run the app in the simulator
3. Drag the OPML file into the simulator
4. ✅ Verify you are subscribed to Hello from the Magic Tavern
5. On your real device running the current App Store build, go to: https://www.icloud.com/shortcuts/0d1cf7d3dc9f4cda9c01d8cd58849331
6. Install the Shortcut
7. Tap on it to run it
8. ✅ Verify you are subscribed to Hello from the Magic tavern

## Checklist
